rts/Sim/MoveTypes/MoveDefHandler.h

```diff
@@ -143,6 +143,10 @@ struct MoveDef {
 	int xsize = 0, xsizeh = 0;
 	int zsize = 0, zsizeh = 0;
 
+	// Apply additional collision boundary in elmos to keep units separated.
+	// The larger of two units' separation distance will be applied.
```

Comment:
would be good to explicitly list the full set of differences between:
* a movedef of size X and a separation of Y
* a movedef of size X+Y and a separation of 0

the existing comment seems to imply the separation doesn't stack, and the issue that prompted the change would suggest that the separation doesn't apply to buildings. are there any others?

Reply:
I've expanded the comments here to help make clear what this field is used for.